set -e errors on ppc64le since no git and no repo
--
--- a/third_party/libvpx/generate_gni.sh
+++ b/third_party/libvpx/generate_gni.sh
@@ -371,7 +371,6 @@
 }
 
 # Fetch the latest tags; used in creating vpx_version.h.
-git -C "${LIBVPX_SRC_DIR}" fetch --tags
 
 find_duplicates
 
